plugin/pkg/scheduler/schedulercache/cache_test.go

```diff
@@ -751,13 +788,18 @@ func benchmarkExpire(b *testing.B, podNum int) {
 	}
 }
 
-func makeBasePod(nodeName, objName, cpu, mem string, ports []v1.ContainerPort) *v1.Pod {
+func makeBasePod(nodeName, objName, cpu, mem, oir string, ports []v1.ContainerPort) *v1.Pod {
 	req := v1.ResourceList{}
 	if cpu != "" {
 		req = v1.ResourceList{
 			v1.ResourceCPU:    resource.MustParse(cpu),
 			v1.ResourceMemory: resource.MustParse(mem),
 		}
+		if oir != "" {
+			name := v1helper.OpaqueIntResourceName(strings.Split(oir, ":")[0])
+			quantity := resource.MustParse(strings.Split(oir, ":")[1])
```

Comment:
@xiangpengzhao since oir string is the part of the test case which are hard-coded values, it makes little sense to me to add len check. It looks to me like testing the test case. Thoughts?